pilot/pkg/serviceregistry/kube/controller/ambientindex.go

```diff
@@ -629,7 +651,7 @@ func (c *Controller) setupIndex() *AmbientIndex {
 			updates := handlePod(oldObj, newObj, false)
 			if len(updates) > 0 {
 				c.opts.XDSUpdater.ConfigUpdate(&model.PushRequest{
-					Full:           false,
+					Full:           true,
```

Comment:
I'm surprised I needed the full push for the update and delete - but not during the add?